Expand description of other hCoV (#624)
greenelab/covid19-review: content/02.introduction.md

An effective public health response involves response coordination, disease surveillance, intervention monitoring, risk communication, and health education (including the containment of “infodemics” of false information) [@isbn:978-92-4-156553-0].
Epidemiology and public health intersect with the topics addressed in this manuscript because they both inform and benefit from relevant biotechnological developments.
For example, the development of accurate and fast diagnostic testing is relevant to test, trace, and isolate strategies for containment, and public education will be critical to deploying a vaccine once one is developed.
The present analysis focuses less on human and social factors and more on the basic biology of infection, diagnosis, and recovery, but these areas are inextricable in understanding and responding to the COVID-19 pandemic.

### Iniital Characterization

The first genome sequence of the virus was identified on January 3, 2020 and was used to determine that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].

> Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7] or from BALF, throat swabs, or isolates of the virus cultured from BALF [@doi:10/ggjr43].

Reviewer: This is clearer now :) yes 5 months has flown by its been a crazy year...

A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East Respiratory Syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with a basic reproduction number of one [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
Subsequently, two additional coronaviruses were identified that infect humans [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was identified in a 7-month-old infant and then identified in clinical specimens collected from seven additional patients, five of whom were infants (younger than 1 year) and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].

> The COVID-19 pandemic is thus associated with the seventh hCoV to be identified and the fifth since the turn of the millenium, though additional hCoVs in circulation may remain undetected.

Author: ```suggestion
The COVID-19 pandemic is thus associated with the seventh hCoV to be identified and the fifth since the turn of the millennium, though additional, undetected hCoVs are likely to be in circulation.
```
Thank you for this suggestion! Does this rephrasing work? I definitely don't want this section to make it sound like the coronaviruses are suddenly a bigger problem, as I suspect it's just that we've figured out how to find them now.

> The COVID-19 pandemic is thus associated with the seventh hCoV to be identified and the fifth since the turn of the millenium, though additional hCoVs in circulation may remain undetected.

Reviewer: I agree, looks good to me

For example, the development of accurate and fast diagnostic testing is relevant to test, trace, and isolate strategies for containment, and public education will be critical to deploying a vaccine once one is developed.
The present analysis focuses less on human and social factors and more on the basic biology of infection, diagnosis, and recovery, but these areas are inextricable in understanding and responding to the COVID-19 pandemic.

> ### Inital Characterization

Reviewer: ```suggestion
### Initial Characterization
```
